framework/log/LoggerInterface.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\log;
+
+/**
+ * LoggerInterface defines a Logger that records logged messages in memory and sends them to different targets.
+ *
+ * A Logger instance can be accessed via `Yii::getLogger()`.
+ * For more details and usage information on Logger, see the [guide article on logging](guide:runtime-logging).
+ *
+ * @author Nelson J Morais <njmorais@gmail.com>
```

Comment:
API was originally authored by Qiang. I think his authoring copyright should not be removed.

Reply:
Done. 👍